testing/citest/tests/kube_v2_cache_test.py

```diff
@@ -0,0 +1,255 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Integration test to see if the image promotion process is working for the
+Spinnaker Kubernetes V2 integration.
+"""
+
+# Standard python modules.
+import sys
+import random
+import string
+import json
+
+# citest modules.
+from citest.service_testing import HttpContractBuilder
+from citest.service_testing import NoOpOperation
+import citest.kube_testing as kube
+import citest.json_contract as jc
+import citest.json_predicate as jp
+import citest.service_testing as st
+
+# Spinnaker modules.
+import spinnaker_testing as sk
+import spinnaker_testing.gate as gate
+import spinnaker_testing.frigga as frigga
+import citest.base
+
+ov_factory = jc.ObservationPredicateFactory()
```

Comment:
Not really a comment specific to this PR, but I see a lot of tests including this. We may want to just add this as a property to the `SpinnakerTestScenario` rather than have everyone declare it as a constant.

Reply:
Good call, will start to look at wider scale cleanup as I rename some tests later.